CHANGELOG.md

```diff
@@ -81,6 +81,7 @@ The `--master_check_interval` flag is removed from `logsigner`.
 ### Performance
 
 The performance of `SetLeaves` requests on the Map has been slightly improved.
+The performance of `GetConsistencyProof` requests has been improved.
```

Comment:
Maybe mention that this is for MySQL (and pgsql) specifically?

Reply:
Done.